src/astro/sql/parsers/sql_directory_parser.py

```diff
@@ -6,22 +7,91 @@
 from airflow.decorators.base import get_unique_task_id
 from airflow.decorators.task_group import task_group
 from airflow.exceptions import AirflowException
+from airflow.models.dag import DagContext
 from airflow.models.xcom_arg import XComArg
 
 from astro.sql.operators.sql_decorator import SqlDecoratedOperator
 from astro.sql.table import Table, TempTable
 
 
+def get_path_for_render(path):
+    """
+    Generate a path using either the DAG's template_searchpath with a relative path, or with the relative path
+    to the DAG file
+
+    :param path:
+    :return:
+    """
+    template_path = DagContext.get_current_dag().template_searchpath
+    if template_path:
```

Comment:
@dimberman in the original ticket description (#123 ), we had:
>  If both `aql.render` path and the template_searchpath are declared, the priority should be given to the path value.

Why did you change this behavior?
Also, since  @mag3141592 was the one who reported this originally, I think it would be great to have her reviewing the PR/feature.

Reply:
@tatiana because choosing one or the other doesn't match the existing behavior. 

If you look at [this example](https://stackoverflow.com/questions/50761084/using-airflow-template-files-and-template-searchpath-in-google-cloud-composer), the template_searchpath is treated as a base directory, not as the whole path. People still point to the file or subdirectory they want to use.

Reply:
@tatiana I discussed with Maggie and have changed the ticket to reflect our discussion.